skptesting/run_fadein_test.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 function run_test() {
-  GO111MODULE=on go test ./loadbalancer -run="$1" -count=1 -v | awk '/fadein_test.go:[0-9]+: CSV/ {print $3}'
+  GO111MODULE=on go test ./proxy -run="$1" -count=1 -v | awk '/fadein_internal_test.go:[0-9]+: CSV/ {print $3}'
```

Comment:
you can also delete `GO111MODULE=on`

Reply:
It was used to enable gomodules in Go 1.11

Reply:
https://github.com/zalando/skipper/pull/2878